src/components/tables/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ParameterizedRouteBases } from '../../App';
+import { Tag, Typography } from 'antd';
+
+const { Link } = Typography;
+
+export const DESCEND_ORDER: 'descend' | 'ascend' | null | undefined = 'descend';
+
+export function dateSorter(a: Date, b: Date): number {
+  return new Date(a).valueOf() - new Date(b).valueOf();
+}
+
+export function renderSiteIdAsLink(siteId: number): JSX.Element {
```

Comment:
⛏️ These should be React components. The reason why you shouldn't return plain old JSX like this is because it messes with how React deals with updating the virtual DOM. I recommend this instead:
```ts
const SiteLink: React.FC<...> = ({siteId}) => {
```

Reply:
This was originally to avoid the `react/display-name` ESLint error when setting up the table columns. Created the React components and instead avoided the error by using JSX style API for the tables instead.